packages/tailwindcss/src/index.ts

```diff
@@ -208,15 +212,40 @@ async function parseCss(
         throw new Error('`@source` cannot be nested.')
       }
 
+      let inline = false
+      let not = false
```

Comment:
```suggestion
      let not = false
      let inline = false
```
🙈

Not required, but maps to the order of `@source not inline('path')`